test/parallel/test-cluster-disconnect-exitedAfterDisconnect-exit.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
The file seems testing the exit without calling the `.disconnect()`? Such that if the name `test-cluster-no-disconnect-exitedAfterDisconnect` would be better?

Reply:
Went with this name, seeing the general naming convention of the other tests for this module

Reply:
`test-cluster-no-disconnect-exitedAfterDisconnect` should be in the convention IMHO.